app/src/androidTest/java/ch/epfl/cs311/wanderwave/model/BeaconConnectionTest.kt

```diff
@@ -33,7 +33,7 @@ public class BeaconConnectionTest {
 
   @Test
   fun testAddAndGetItem() = runBlocking {
-    withTimeout(30000) { // Timeout after 5000 milliseconds (20 seconds)
+    withTimeout(30000) { // Increase the timeout to 30 seconds
```

Comment:
I think having timeouts is a bad practice and lead to unexpected workflow fails

Reply:
When dealing with backend I think it's a good thing. I just changed the default timeout from 20 to 30